R/edi_qaqc_combine.R

```diff
@@ -0,0 +1,396 @@
+qaqc_edi_combine <- function(realtime_file,
+                               qaqc_file,
+                               config,
+                               input_file_tz){
+
+
+  # realtime_file = L1 FILE STORED ON GITHUB
+  # qaqc_file = EXISTING FILE STORED ON EDI
+
+  edi_exist <- TRUE
+
+  #d1 <- read.csv(realtime_file, na.strings = 'NA', stringsAsFactors = FALSE)
+  d1 <- realtime_file
+
+  salt_quantiles <- quantile(((0.001 * d1$EXOSpCond_uScm_1) ^ 1.08), c(0.0025,0.975), na.rm = TRUE)
```

Comment:
This seems like a qaqc step.  Should it be in the EDI qaqc script?  Does it not also apply both to the EDI and L1 files?